modules/network/src/main/java/org/apache/ignite/network/Network.java

```diff
@@ -17,50 +17,15 @@
 
 package org.apache.ignite.network;
 
-import java.util.Arrays;
-import java.util.Collections;
-import org.apache.ignite.network.message.MessageMapperProvider;
-
 /**
- * Entry point for network module.
+ * Entry point for the network module.
  */
-public class Network {
-    /** Message mapper providers, messageMapperProviders[message type] -> message mapper provider for message with message type. */
-    private final MessageMapperProvider<?>[] messageMapperProviders = new MessageMapperProvider<?>[Short.MAX_VALUE << 1];
-
-    /** Message handlers. */
-    private final MessageHandlerHolder messageHandlerHolder = new MessageHandlerHolder();
-
-    /** Cluster factory. */
-    private final NetworkClusterFactory clusterFactory;
-
-    /**
-     * Constructor.
-     * @param factory Cluster factory.
-     */
-    public Network(NetworkClusterFactory factory) {
-        clusterFactory = factory;
-    }
+public interface Network {
```

Comment:
What about renaming it to `Networking`? It is not ideal either but sounds a bit better to me than plain `Network`.

Reply:
We've had a discussion and decided to change it to `ClusterService`